conf/test_no_crai.config

```diff
@@ -5,22 +5,24 @@
         Defines input files and everything required to run a fast and simple pipeline test.
 
         Use as follows:
-                nextflow run nf-core/bamtofastq -profile test,<docker/singularity> --outdir <OUTDIR>
+                nextflow run nf-core/bamtofastq -profile test,test_no_crai,<docker/singularity> --outdir <OUTDIR>
 
 ----------------------------------------------------------------------------------------
 */
 
+
+
 params {
-        config_profile_name = 'Test profile'
-        config_profile_description = 'Minimal test dataset to check pipeline function'
+    config_profile_name = 'Test no crai profile'
+    config_profile_description = 'Minimal test dataset to check pipeline function'
 
-        // Limit resources so that this can run on GitHub Actions
-        max_cpus = 2
-        max_memory = '6.GB'
-        max_time = '48.h'
+    // Limit resources so that this can run on GitHub Actions
+    max_cpus = 2
+    max_memory = '6.GB'
+    max_time = '48.h'
```

Comment:
is this needed?

Reply:
I think you set up the `.github/workflows/ci.yml` differently than I did and that's why it's currently not loading the test profile correctly. I'm trying to figure out how you defined the [tests](https://github.com/nf-core/sarek/blob/c87f4eb694a7183e4f99c70fca0f1d4e91750b33/.github/workflows/ci.yml#L88)